admin_manual/issues/impersonate_users.rst

```diff
@@ -0,0 +1,49 @@
+===================
+Impersonating Users
+===================
+
+Sometimes you may need to use your ownCloud installation as another user, whether to help users debug an issue or to get a better understanding of what they see when they use their ownCloud account.
+The ability to do so is a feature in ownCloud 10.0, via a `Marketplace` app called "**Impersonate**". 
```

Comment:
For your understanding: The marketplace is our new way to distribute apps/extensions. All apps will be on the marketplace (sooner or later).